scripts/stablex/utilities.js

```diff
@@ -29,6 +31,12 @@ const addTokens = async function (token_addresses, web3, artifacts) {
   }
 }
 
+const closeAuction = async (instance) => {
```

Comment:
This method is now implemented twice. Once in `test/stablex/stablex_utils` and here. Can we not merge the implementations?

Reply:
This is temporary as part of a "Part I" refactoring. The intention is to finish the refactoring with "Part II" and deal with this.